samples/test/kind-config.yaml

```diff
@@ -0,0 +1,5 @@
+kind: Cluster
```

Comment:
Add TODO  - remove this file when ImageVolume is GA or accessible outside of feature gate.

Reply:
Done.